web/src/containers/Home.js

```diff
@@ -23,6 +27,9 @@ const StyledTableHeader = styled.div`
     color: white;
     background-color: #036;
   }
+  .hide {
+    display: none !important;
+  }
 `;
 
 export class Home extends Component {
```

Comment:
not a big deal but because this component only seems to be using one piece of state and _no_ lifecycle methods (componentDidMount etc). This component could be converted to a functional component and leverage `useState`

Reply:
thanks for pointing out, I might be using componentDidMount for this page soon cuz I just  realized that it's not refreshing when coming back to the page...

Reply:
useEffect maybe the best use of that since it will run on every re render. DidMount only runs the first time the component has mounted.